base/domain-sot.yml

```diff
@@ -382,6 +382,11 @@ responses:
         O entse hantle ka ho itlamo ka bophelo bo botle ba hao! 👏
       claim: release
 
+  utter_planning_to_test:
+    - text: |
+        Well done for planning to get your test! 👏
```

Comment:
english?

Reply:
We still waiting for translation, this is a placeholder